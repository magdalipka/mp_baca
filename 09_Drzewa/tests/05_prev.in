1
8
ENQUE 35 Andrzej Wolny
ENQUE 30 Andrzej Inny
ENQUE 40 Iwona Wolny
ENQUE 45 Iwona Inny
PREV 30
PREV 35
PREV 40
PREV 45
