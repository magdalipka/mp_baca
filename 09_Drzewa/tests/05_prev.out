Zestaw 1
PREV 30: BRAK
PREV 35: 30 - Andrzej Inny
PREV 40: 35 - Andrzej Wolny
PREV 45: 40 - Iwona Wolny
